src/packages/@ncigdc/components/analysis/SetOperations/OpsTable.js

```diff
@@ -32,93 +33,117 @@ const ActionsTd = compose(
     const saveSetDisabled = count > MAX_SET_SIZE;
 
     return (
-      <Td style={{ textAlign: 'right', opacity: hide ? 0 : 1 }}>
-        <Tooltip
-          Component={
-            saveSetDisabled
-              ? `Set Operation can not be saved because it is over the limit of ${MAX_SET_SIZE}`
-              : 'Save selection as new set'
-          }
-          style={{ marginRight: 5 }}
-        >
-          <i
-            style={
-              saveSetDisabled
-                ? { color: theme.greyScale4 }
-                : { cursor: 'pointer', color: 'rgb(37, 94, 153)' }
-            }
-            className="fa fa-save"
-            onClick={() => {
-              if (saveSetDisabled) return;
-              dispatch(
-                setModal(
-                  <SaveSetModal
-                    title={`Save selection as new set`}
-                    total={count}
-                    filters={filters}
-                    type={type}
-                    displayType={type}
-                    CreateSetButton={CreateSetButton}
-                    setName="My Set"
-                  />,
-                ),
-              );
-            }}
-          />
-        </Tooltip>
-        <Tooltip Component="Export as TSV" style={{ marginRight: 5 }}>
-          <DownloadButton
-            className="test-download-set-tsv"
-            style={iconButton}
-            endpoint={`${type}s`}
-            activeText="" //intentionally blank
-            inactiveText="" //intentionally blank
-            altMessage={false}
-            setParentState={() => {}}
-            active={false}
-            filters={filters}
-            extraParams={{ format: 'tsv' }}
-            fields={downloadFields[type]}
-            filename={fileName
-              .replace(/∩/g, 'intersection')
-              .replace(/∪/g, 'union')}
-          />
-        </Tooltip>
-        {type === 'case' && (
-          <CreateSetButton
-            filters={filters}
-            onComplete={setId => {
-              push({
-                pathname: '/repository',
-                query: {
-                  filters: stringifyJSONParam({
-                    op: 'AND',
-                    content: [
+      <Td style={{ textAlign: 'right' }}>
+        {hide ? (
```

Comment:
i'm not normally a fan of multiple returns but I think because it's wrapping such a big chunk of js might be easier to follow if

`if (hide) return <Td ... />`

Reply:
changing it to `!hide &&`